TransformationSystem/Agent/TransformationAgent.py

```diff
@@ -223,12 +223,12 @@ def _execute( self, threadID ):
         self._logVerbose( "%d transformations still in queue" % len( self.transInQueue ) )
     return S_OK()
 
-  def processTransformation( self, transDict, clients, active = True ):
+  def processTransformation( self, transDict, clients ):
     """ process a single transformation (in transDict)
     """
-
+    method = 'processTransformation'
     transID = transDict['TransformationID']
-    replicateOrRemove = transDict['Type'].lower() in ( 'replication', 'removal' )
+    forJobs = transDict['Type'].lower() not in ( 'replication', 'removal' )
 
     # First get the LFNs associated to the transformation
     transFiles = self._getTransformationFiles( transDict, clients, replicateOrRemove = replicateOrRemove )
```

Comment:
As per https://landscape.io/github/DIRACGrid/DIRAC/pulls/3237, you forgot to updates line 234 and 247